index.js

```diff
@@ -15,7 +15,7 @@ export default class ElevatedView extends React.Component {
 
     if (Platform.OS === 'android') {
       return (
-        <View elevation={elevation} style={style} {...otherProps}>
+        <View style={[{ elevation, backgroundColor: 'white' }, style ]} {...otherProps}>
```

Comment:
lets remove `backgroundColor: white`, resulting in 

```
<View style={[{ elevation }, style]} {...otherProps}>
```

Reply:
without the backgroundColor the shadow doesn't work at all on Android

Reply:
Ok, sounds good. Should we leave the `elevation={elevation}` prop around as well so that it is backwards compatible with other RN versions?

Reply:
done